File: NET/API/Treehopper/UartMode.cs
namespace Treehopper
{
    /// <summary>
    /// Whether to use UART or OneWire mode
    /// </summary>
    public enum UartMode
    {
        /// <summary>
        /// The module is operating in UART mode
        /// </summary>
        Uart,

        /// <summary>
        /// The module is operating in OneWire mode. Only the TX pin is used.
        /// </summary>
        OneWire
    }
}

File: Java/api/treehopper/src/main/java/io/treehopper/enums/UartMode.java
package io.treehopper.enums;

/**
 * UART mode
 */
public enum UartMode
{
    /**
     * The module is operating in UART mode
     */
    Uart,

    /**
     * The module is operating in OneWire mode. Only the TX pin is used.
     */
    OneWire
}